Editor/Utility/BaseEditor.cs

```diff
@@ -92,6 +92,24 @@ public override void OnInspectorGUI()
             DrawRemainingPropertiesInInspector();
         }
 
+        /// <summary>
+        /// Draw inside the scene view here. Base implementation calls DrawSceneTools.
+        /// </summary>
+        protected virtual void OnSceneGUI()
+        {
+            DrawSceneTools();
+        }
+
+        /// <summary>
+        /// Implement your scene handles for your components and extensions here.
+        ///
+        /// Don't forget to register/unregister your components and extensions in their
+        /// editor's OnEnable and OnDisable functions with CinemachineSceneToolUtility.RegisterTool/UnregisterTool. 
+        /// </summary>
+        /// <param name="guideLinesColor">Color of guidelines</param>
+        /// <param name="defaultColor">Color of handles, when user is not using them</param>
+        protected virtual void DrawSceneTools() {}
+
```

Comment:
I think this does not belong in this class, because it's too specific to vcam inspectors, and because it will break existing derived classes thanks to the new virtual OnSceneGUI.

Instead, make a new class that inherits this and add the OnSceneGUI and DrawSceneTools methods there.  Have the tools-aware inspectors inherit from that (safe to do since they are all internal)